src/components/Calculator.js

```diff
@@ -1,49 +1,59 @@
 /* eslint-disable react/button-has-type */
```

Comment:
- [ ] Please remove the eslint disable comment here. It disables linter checks which degrade the quality of your code.

Reply:
This change has been done well